lib/chef-dk/policyfile_services/export_repo.rb

```diff
@@ -241,7 +243,7 @@ def create_client_rb
   puts("!" * 80)
   puts(<<-MESSAGE)
 This Chef Repo requires features introduced in Chef 12.7, but you are using
-Chef #{Chef::VERSION}. Please upgrade to Chef 12.7 or later.
+Chef \#{Chef::VERSION}. Please upgrade to Chef 12.7 or later.
```

Comment:
I want it to get rendered as `#{string_interpolation}` so that it will get interpolated when the config gets evaluated instead of when it gets generated. Otherwise, it will end up being hard-coded to the version of chef that is active inside ChefDK when you run `chef export` instead of the version of chef that is reading the config file.